topics/ecology/tutorials/openrefine_gbif/tutorial.md

```diff
@@ -0,0 +1,327 @@
+---
+layout: tutorial_hands_on
+
+title: Cleaning GBIF data using OpenRefine
+zenodo_link: ''
+questions:
+- How can I use OpenRefine to clean data?
+- How do I check and clean biodiversity data using OpenRefine?
+
+objectives:
+- Use OpenRefine faceting functionnalities to apply mass editing and manage duplicates
+- Use OpenRefine clustering and filtering functionnalities to edit, transform data notably using regular expression
+- Use OpenRefine to apply API services on your data
+time_estimation: 0h45m
+key_points:
+- OpenRefine is a powerfull tool, with many functionnalities to check, clean and enrich your data
+contributions:
+    authorship:
+        - yvanlebras
+        - sophiepamerlon
+        - laurarussell
+        - gbiftraining
+    funding:
+        - pndb
+
+
+---
+
+
+In this tutorial we will use OpenRefine tool to clean occurence records retrieved from GBIF.
```

Comment:
you sometimes write OpenRefine, and sometimes Openrefine (without capitalization of the R), best to pick one and be consistent